Makefile

```diff
@@ -2,16 +2,21 @@ build:
 	@yarn turbo run build
 build-sequential:
 	@yarn turbo run --concurrency 1 build
-python-deps:
+
+venv-setup:
+	@if uv --version >/dev/null 2>&1; then \
+		[ -f .venv/bin/python ] || uv venv --python-preference only-system --python 3.10 --seed .venv; \
+	else \
+		[ -f .venv/bin/python ] || python3 -m venv .venv; \
+	fi
+
+venv-activate:
+	@. .venv/bin/activate
```

Comment:
This does not do what you expect it to do. Note that `source` or `.` are bash builtin commands, so they only work if you're running it inside bash itself.